client-app/pages/account/edit-quote.vue

```diff
@@ -422,7 +422,7 @@ async function submit(): Promise<void> {
 
   await submitQuote(quote.value!.id, comment.value || "");
 
-  void router.replace({ name: "Quotes" });
+  router.replace({ name: "Quotes" }).catch(Logger.error);
```

Comment:
return void. If no need result we place it before promise.

Reply:
reverted changes with "void"